libs/template/reader_test.go

```diff
@@ -0,0 +1,120 @@
+package template
+
+import (
+	"context"
+	"io"
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	"github.com/databricks/cli/internal/testutil"
+	"github.com/databricks/cli/libs/cmdio"
+	"github.com/databricks/cli/libs/flags"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestBuiltInReader(t *testing.T) {
+	exists := []string{
+		"default-python",
+		"default-sql",
+		"dbt-sql",
+	}
+
+	for _, name := range exists {
+		t.Run(name, func(t *testing.T) {
+			r := &builtinReader{name: name}
```

Comment:
The reader isn't closed in this test. Should it?

Reply:
Close / Cleanup is a no-op, so it does not matter. It's fine to not call it. I've also modified the signature of the Cleanup function to not return an error since we don't check that error in the main `init` command in `init.go`.